### PR TITLE
Update ec2 best case

### PR DESCRIPTION
**Pull request by Harshul1310**
Tenancy check for dedicated server is been created under file name dedicated_vpc-used

### CODE REVIEW

File: src/analyzers/aws/ec2/dedicated_ec2_instance.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                    instanceAnalysis.action= "Amount to be paid is +++";
+                }
+                allRegionsAnalysis[region].push(instanceAnalysis);
+            }
+        }
+        dedicated_instance_used.regions = allRegionsAnalysis;
+        return { dedicated_instance_used };
+    }
+
+    private getdedicatedinstance(instance: any[]) {
```
Comment (paragpoddar) [inline src/analyzers/aws/ec2/dedicated_ec2_instance.ts]: use camelcase. change getdedicatedinstance to getDedicatedInstance.

File: src/analyzers/aws/ec2/dedicated_ec2_instance.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        if (!instance) {
+            return [];
+        }
+        return instance.filter((instances) => {
+            return instances.Isdedicated;
+        });
+    }
+
+    private isinstancesExist(instance, instancesId) {
```
Comment (paragpoddar) [inline src/analyzers/aws/ec2/dedicated_ec2_instance.ts]: isinstancesExist --> isInstancesExist

File: src/analyzers/aws/iam/index.ts [hunk 3]
```diff
@@ -1,9 +1,11 @@
 export * from "./admins_count";
 export * from "./groups_used";
+export * from "./instances_profiles_used"
```
Comment (paragpoddar) [inline src/analyzers/aws/iam/index.ts]: This file is missing.

Comment (Harshul1310) [reply src/analyzers/aws/iam/index.ts]: Ok

**Comment by paragpoddar** [comment]
I am not seeing any data for dedicated instances used in report. Also change the class name "DedicatedTenUsedEC2InstancesAnalyzer" into "DedicatedEC2InstancesUsedAnalyzer". Please follow other class name.

**Pull request merged**